pkgs/applications/gis/tunnelx/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, jdk
+, jre
+, fetchgit
+, survex
+, makeWrapper
+, bash
+, coreutils
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
you don't need the rec here since you aren't reusing any attribute.